packages/http/src/getHttpOperations.ts

```diff
@@ -27,7 +27,9 @@ const httpAndFileResolver = new Resolver({
 
 export async function getHttpOperationsFromResource(file: string): Promise<IHttpOperation[]> {
   const isRemote = /^https?:\/\//i.test(file);
-  const fileContent = isRemote ? await fetch(file).then(d => d.text()) : fs.readFileSync(file, { encoding: 'utf8' });
+  const fileContent = await (isRemote
```

Comment:
Out of curiosity, does this change fix anything?

Reply:
No, it just uses the fs promise API instead of the sync one